rayon-core/Cargo.toml

```diff
@@ -12,10 +12,6 @@ readme = "README.md"
 keywords = ["parallel", "thread", "concurrency", "join", "performance"]
 categories = ["concurrency"]
 
-[features]
-default = ["tlv"]
-tlv = []
```

Comment:
Isn't removing a feature backwards-incompatible?

Reply:
Probably, `rustc-rayon` doesn't provide any guarantees though.

Reply:
Still, might as well bump to "0.2.0" I would think?

Reply:
(At least that's what I usually do)